composer.json

```diff
@@ -1,15 +1,25 @@
 {
     "name": "passageidentity/passage-php",
-    "description": "Passage's management API to manage your Passage apps and users.",
-    "version": "0.4.0",
+    "description": "Passkey Complete for PHP - Integrate into your PHP API or service to enable a completely passwordless standalone auth solution with Passage by 1Password.",
+    "version": "0.4.1",
```

Comment:
This may fix the badge problem. Up to you if you want to test it or just leave it for the backend sdk improvement work.
```suggestion
```